src/MSBuild/MSBuild/Microsoft.Build.CommonTypes.xsd

```diff
@@ -6173,5 +6173,11 @@ elementFormDefault="qualified">
       </xs:complexContent>
     </xs:complexType>
   </xs:element>
+  
+  <xs:element name="InvariantGlobalization" type="msb:boolean" substitutionGroup="msb:Property">
+    <xs:annotation>
+      <xs:documentation><!-- _locID_text="InvariantGlobalization" _locComment="" -->Indicates whether an app runs in globalization-invariant mode without access to culture-specific data and behavior.</xs:documentation>
```

Comment:
```suggestion
      <xs:documentation><!-- _locID_text="InvariantGlobalization" _locComment="" -->Determines whether an app runs in globalization-invariant mode without access to culture-specific data and behavior.</xs:documentation>
```
imo it's clearer

Reply:
I did this to follow the pattern in other documented boolean-style properties. They all use "Indicates". The notable excpetion being `Using_Static` which starts with "Determines."